eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration/ConfigResolver.php

```diff
@@ -165,6 +171,10 @@ public function hasParameter($paramName, $namespace = null, $scope = null)
      */
     public function getParameter($paramName, $namespace = null, $scope = null)
     {
+        if (!$this->container instanceof ContainerBuilder && $this->siteAccess->matchingType === 'uninitialized') {
+            $this->tooEarlyLoadedList[$paramName][] = $this->extractServiceName();
```

Comment:
I think we should do this only when `%kernel.debug%` is `true`. 
But... If you think this is valuable for production also, then here should be just a hint to check it in `dev` (or just debug) mode for more details. `debug_backtrace` used in `extractServiceName` is not something I'd use in production code.

Reply:
I agree, and by default console is in dev + debug so people will still see it where it counts

Reply:
I have thought about that, but depending on the consumer's integration system, it is very likely that some issues will only show in up in prod, and will be hard to reproduce in dev.

In production, you can handle warnings any way you want. I'd keep in both modes.

Reply:
Do you have examples of such case?

Reply:
Closing this thread, feel free to open new comment @bdunogier